docs/BacwardCompatibility.md

```diff
@@ -0,0 +1,133 @@
+- [Policy Manager Interface](#data-catalog-and-policy-manager-connector-interfaces)
+- IT Configuration Policies
+- IT attributes
+- ....
+
+
+## Kubernetes Custom Resource Versions and Upgrade Plan
+
+### Versions
+Fybrik for its Kubernetes API objects (CRDs) uses API versioning similar to one used by
+[Kubernetes](https://kubernetes.io/docs/reference/using-api/#api-versioning).
+- Alpha
+    - The version names contain alpha (for example, v1alpha1).
+    - The software is not stable may contain bugs.
+    - The support for a feature may be dropped at any time without notice.
+    - The API may change in incompatible ways in a later software release without notice.
+- Beta
+    - The version names contain beta (for example, v2beta3).
+    - The software is well tested. Enabling a feature is considered safe.
+    - The support for a feature will not be dropped, though the details may change.
+    - The schema and/or semantics of objects may change in incompatible ways in a subsequent beta or stable release.
+      When this happens, migration instructions are provided. Schema changes may require deleting, editing, and re-creating
+      API objects. The editing process may not be straightforward. The migration may require downtime for applications that
+      rely on the feature.
+- Stable
+    - The version name is vX where X is an integer.
+    - The stable versions of features appear in released software for many subsequent versions.
+### Relations between CRD changes and Fybrik releases
+Based on the above definitions, the relation between CRD versions and Fybrik versions is unidirectional. Which means,
+CRD API changes requires at least minor Fybrik release, on the other hand not each Fybrik release requires a new CRD
+version.
+
+### Upgrade Plan
+The CustomResourceDefinition API provides a workflow for introducing and upgrading to new versions of a 
+CustomResourceDefinition. For more information about 
+[serving multiple versions](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#serving-multiple-versions-of-a-crd) of your CustomResourceDefinition 
+and migrating your objects from one version to another see 
+[Versions in CustomResourceDefinitions](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/)
+
+The first step of adding a new version is to "pick a conversion strategy. Since custom resource objects need to be able 
+to be served at both versions, that means they will sometimes be served at a different version than their storage 
+version. In order for this to be possible, the custom resource objects must sometimes be converted between the version 
+they are stored at and the version they are served at. If the conversion involves schema changes and requires custom 
+logic, a conversion **Webhook** should be used. If there are no schema changes, the default **None** conversion strategy may 
```

Comment:
Several sentences down you state that there is a limitation to using webhooks, so why state here that they should be used?  Are we going to support both webhook based conversions as well as home built migration tools?

Reply:
It is the Kubernetes way to support API migration. And here I just mention that the conversation should be set to "Webhook". It allows only 2 possible values: "none" and "Webhook". Below, I explain that we cannot use Webhooks.

Reply:
OK, will change the statement